pom.xml

```diff
@@ -631,12 +631,12 @@
     <developer>
       <name>Gilles Sadowski</name>
       <id>erans</id>
-      <email>erans at apache dot org</email>
+      <email>erans@apache.org</email>
     </developer>
     <developer>
       <name>Matt Juntunen</name>
       <id>mattjuntunen</id>
-      <email>mattjuntunen at apache dot org</email>
+      <email>mattjuntunen@apache.org</email>
```

Comment:
Please do not change the email formatting here or above. This becomes part of the public website for the project (see https://commons.apache.org/proper/commons-geometry/commons-geometry-core/team.html) and it is up to the individual developer to decide how their email should be displayed.

Reply:
done